ConwaysGameOfLife.Tests/LocationShould.cs

```diff
@@ -57,5 +29,61 @@ public void WhenTwoLocationsInstantiated_NotBeEqual(int x1, int y1, int x2, int
 
             Assert.Equal(expected, result);
         }
+
+        [Fact]
+        public void WhenTwoListsOfLocationsSameOrderInstantiated_BeEqual()

```

Comment:
No need to say `Instantiated` as we're not really testing the constructor for a class you've written
```suggestion
        public void GivenTwoListsOfLocationsSameOrder_BeEqual()
```